index.js

```diff
@@ -66,8 +74,6 @@ class PD {
       server: true,
     });
     await discovery.flushed();
```

Comment:
you should prop not wait for this either, no need in a module like this but up to you.